its/ruling/src/test/resources/expected/python-S5644.json

```diff
@@ -1,49 +1,53 @@
 {
```

Comment:
Unfortunately changes here are FPs (documented in the tests for S5644) due to a wrong stub annotation for `type.__getstate__`